plus/impl/src/main/java/org/sakaiproject/plus/impl/PlusServiceImpl.java

```diff
@@ -47,13 +51,18 @@
 
 import org.sakaiproject.user.api.User;
 import org.sakaiproject.site.api.Site;
+import org.sakaiproject.site.api.SiteService;
 import org.sakaiproject.event.api.Event;
 
 import org.sakaiproject.lti.api.LTIException;
 import org.sakaiproject.component.api.ServerConfigurationService;
 import org.sakaiproject.grading.api.AssessmentNotFoundException;
 import org.sakaiproject.grading.api.GradingService;
 import org.sakaiproject.grading.api.CommentDefinition;
+import org.sakaiproject.exception.IdUnusedException;
+
+import org.sakaiproject.authz.api.SecurityAdvisor;
+import org.sakaiproject.authz.cover.SecurityService;
```

Comment:
Use org.sakaiproject.authz.api.SecurityService;

Reply:
Yes - I made this change.  Thanks.  Too much cutting and pasting of code that used the wrong patterns.